packages/common/src/grpc-service.js

```diff
@@ -161,6 +161,11 @@ function GrpcService(config, options) {
 
   this.grpcMetadata = new grpc.Metadata();
 
+  this.grpcMetadata.add('User-Agent', [
+    config.packageJson.name.replace('@google-cloud', 'gcloud-node'),
```

Comment:
Not sure, that's still being discussed in https://github.com/GoogleCloudPlatform/gcloud-common/issues/179. But without the service, the version would easily conflict with other service versions... although, I suppose if the Bigtable API is getting a request, it could assume that the version number refers to the Bigtable module.